src/Log4netShipper/LogzioAppender.cs

```diff
@@ -38,14 +42,30 @@ private void WriteImpl(LoggingEvent loggingEvent)
 					{"thread", loggingEvent.ThreadName},
 					{"message", loggingEvent.RenderedMessage},
 					{"exception", loggingEvent.GetExceptionString()},
-					{"user", loggingEvent.UserName},
+					// log4net already adds this 'username' property which is added to values bellow
+					//{"user", loggingEvent.UserName},
+					{"processId", ProcessId}
 				};
 
 				foreach (var customField in _customFields)
 				{
 					values[customField.Key] = customField.Value;
 				}
 
+				var properties = loggingEvent.GetProperties();
+				foreach (DictionaryEntry property in properties)
+				{
+					var value = property.Value?.ToString();
+					if (string.IsNullOrWhiteSpace(value))
+						continue;
+
+					var key = property.Key.ToString();
+					key = key.Replace(":", "");
+					key = key.Replace(".", "");
+					key = key.Replace("log4net", "");
+					values["property" + key] = value;
```

Comment:
Well.. collision is possible. I thought would be nice to have a clear separation of these properties. On the other hand its not really needed and can be without 'property' prefix. In some cases it is even more nice instead of 'propertyIdentity' to have 'Identity'. So I leave it for you to make final decision here.